tests/Microsoft.Identity.Test.Integration.netcore/Microsoft.Identity.Test.Integration.NetCore.csproj

```diff
@@ -19,11 +19,10 @@
     <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.9" />
     <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="6.15.0" />
     <PackageReference Include="Microsoft.IdentityModel.Protocols.SignedHttpRequest" Version="6.15.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
-    <PackageReference Include="MSTest.TestAdapter" Version="2.1.0" />
-    <PackageReference Include="MSTest.TestFramework" Version="2.1.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
```

Comment:
Might be worth moving to using Central Package Management https://devblogs.microsoft.com/nuget/introducing-central-package-management/

Reply:
Absolutely, but currently there is something in MSTest that causes it to fail with newer versions... it's weird :)

Reply:
`[MSTest][Discovery][D:\a\1\s\tests\Microsoft.Identity.Test.Unit\bin\Debug\net472\Microsoft.Identity.Test.Unit.dll] Failed to discover tests from assembly D:\a\1\s\tests\Microsoft.Identity.Test.Unit\bin\Debug\net472\Microsoft.Identity.Test.Unit.dll. Reason:Could not load file or assembly 'Microsoft.Identity.Test.Unit' or one of its dependencies. The system cannot find the file specified.` ... not a great experience on the part of mstest. I assume it has to do with multi-TFM test projects.

Reply:
Yeah, i've setup a build on another DevOps instance to see what's going on. Curious to find the cause of this :)

Reply:
Resolved that one. Seems like the TestAdapter for NUnit messed up Test Discovery or something. Pointing the VSTest task to a location without custom Test Adapters for the test seems to work. 

Got another test failing now, which seems weird that updating VSTest could have anything to do with it...
```

 Microsoft.Identity.Test.Integration.HeadlessTests.UsernamePasswordIntegrationTests.ROPC_ADFSv3Federated_Async

Test method Microsoft.Identity.Test.Integration.HeadlessTests.UsernamePasswordIntegrationTests.ROPC_ADFSv3Federated_Async threw exception: 
Microsoft.Identity.Client.MsalUiRequiredException: AADSTS5000811: Unable to verify token signature. The signing key identifier does not match any valid registered keys.
Trace ID: 04d9f884-1ffe-403e-8d67-28130e687d00
Correlation ID: ad8c894a-557f-48c0-b045-c129590c344e
Timestamp: 2022-07-26 08:17:00Z
```

Ideas?

Reply:
Ha... was a temporary error it seems, or someone resolved it